app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class KeywordsController < ApplicationController
+  before_action :ensure_authentication
+  before_action :set_csv_form
+
+  def new
+    render locals: {
+      csv_form: @csv_form
+    }
+  end
+
+  def create
+    save_result = @csv_form.save(create_params)
+
+    if save_result.class == ActiveRecord::Result
```

Comment:
@longnd @gutakk normally the save method return `true or false` we try to keep the FormObject as same as the Active Record.

That was defined in our Compass https://compass.nimblehq.co/development/code-conventions/ruby-on-rails/#forms

Reply:
@gutakk Once you decide to go with Form Object, you can take a look on this blog https://nimblehq.co/blog/lets-play-design-patterns-form-objects ;)

Reply:
```normally the save method return true or false we try to keep the FormObject as same as the Active Record.```

If we need to keep Form Object same as Active Record, how can we receive `returning value` from database insertion (`Keyword.insert_all(bulk_data, returning: %w[id keyword])`)? As I need this value to create sidekiq job.

Reply:
we can assign it to another public attribute of the `FormObject` @gutakk , for example

```
class FormObeject
  attr_reader :keywords

 def save
    ...

    if result.class == ActiveRecord::Result
       @keywords = result
       true
    else
      false
    end
 end
end

# on controller

def create
  if form_obect.save
  else
  end
end
```

Reply:
Thanks for the example
Fixed in 974780e43306845042fd43f163b9598ad614b8f3